changelog.txt

```diff
@@ -1,7 +1,17 @@
-4.3.1
+4.3.2
 =============
+New Features:
+- **MSAL.NET now sends telemetry data to the /token endpoint in regards to the error code of the previous request, if applicable**. This will enable MSAL.NET to determine reliablity across public client application calls.
+
 Bug Fixes:
+- **For iOS 13, iOS broker may or may not return the source application**. The source appplication is used by MSAL.NET to verify the response is coming from broker. MSAL.NET will now also create a nonce to send in the broker request and will verify the same nonce is returned in the broker response in the case of a missing source application. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1375)
```

Comment:
*Ensures that MSAL.NET works fine with brokers in iOS13**. Indeed, on iOS 13, iOS, the broker may or may not return the source application, whcih is used by MSAL.NET to verify that the response is coming from the broker. To maintain secure calls, MSAL.NET will now also create a nonce to send in the broker request and will verify the same nonce is returned in the broker response in the case of a missing source application. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1375) ?

Reply:
issue linked is not the right one

---
In reply to: [325975379](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1392#discussion_r325975379) [](ancestors = 325975379)

Reply:
@henrik-me nice catch. i fixed it (took me two tries, but it's now correct). lol.